podman.spec.rpkg

```diff
@@ -164,7 +164,7 @@ export BUILDTAGS="$BASEBUILDTAGS $(hack/btrfs_installed_tag.sh) $(hack/btrfs_tag
 make docs docker-docs
 
 %install
-PODMAN_VERSION=%{version} %{__make} DESTDIR=%{buildroot} PREFIX=%{_prefix} ETCDIR=%{buildroot}%{_sysconfdir} \
+PODMAN_VERSION=%{version} %{__make} DESTDIR=%{buildroot} PREFIX=%{_prefix} ETCDIR=%{_sysconfdir}/containers \
```

Comment:
I don't think this is correct. I think `ETCDIR` is intended to be `/etc`, `/usr/local/etc`, or something along those lines.

Reply:
See: https://github.com/containers/podman/pull/17914#issuecomment-1491744145

Reply:
I'm sorry, I've looked everywhere and don't see the connection between `ETCDIR` and `nodocker`.

The only place `nodocker` appears is in the [docker wrapper script](https://github.com/containers/podman/blob/3e44a7afed7a5751ef700729f97b9f6e955fcc5e/docker). There is nothing to do with `ETCDIR` anywhere in there.

Perhaps you're thinking of an old iteration of #17791? For a while that one used `ETCDIR` incorrectly, but that is now fixed. I also believe that PR now fixes the erroneous buggy `ETCDIR` invocation in the spec.

@lsm5 here is my recommendation:
1. close this PR
2. go to fedpkg, fix the specfile as follows:
```diff
- ... ETCDIR=%{buildroot}%{_sysconfdir} \
+ ... ETCDIR=%{_sysconfdir} \
```
It may be necessary to apply that change to other fedpkg branches.

I believe that is all that needs to be done.

Reply:
> I'm sorry, I've looked everywhere and don't see the connection between `ETCDIR` and `nodocker`.
> 
> The only place `nodocker` appears is in the [docker wrapper script](https://github.com/containers/podman/blob/3e44a7afed7a5751ef700729f97b9f6e955fcc5e/docker). There is nothing to do with `ETCDIR` anywhere in there.

Well it explicitly mentions /etc in there currently and after 17791 goes in, it will use $ETCDIR/nodocker, so not sure what makes you say that.

> 
> Perhaps you're thinking of an old iteration of #17791? For a while that one used `ETCDIR` incorrectly, but that is now fixed. I also believe that PR now fixes the erroneous buggy `ETCDIR` invocation in the spec.
> 
> @lsm5 here is my recommendation:
> 
> 1. close this PR
> 2. go to fedpkg, fix the specfile as follows:
> 
> ```diff
> - ... ETCDIR=%{buildroot}%{_sysconfdir} \
> + ... ETCDIR=%{_sysconfdir} \
> ```
> 
> It may be necessary to apply that change to other fedpkg branches.
> 
> I believe that is all that needs to be done.

This does break people who already have `/etc/containers/nodocker` in that it will nag people to now create `/etc/nodocker`. But I guess the breakage is trivial enough that we can ask people to create `/etc/nodocker` and move on.

Reply:
Before we continue this conversation, can you please look at #17791 and do a refresh/reload so you can be sure you're looking at the most recent changes as of Friday March 31?

Reply:
ah lol, ok. Tried the latest checkout. works fine. Sorry I spoke on outdated info @edsantiago